travis/psql_log.py

```diff
@@ -15,7 +15,7 @@ def get_psql_conf_files(psql_conf_path=None):
 
 def get_default_log_path(directory=None, filename=None, root_path=None):
     if directory is None:
-        directory = 'pg_log'
+        directory = '/var/log/pg_log'
```

Comment:
I think this dir should be OS agnostic (not always in /var/log) there is a command on systemd or psql command to retreive this.

Reply:
@nhomar by default the logs from pgbadger are stored in that path, also that  is in case nothing is specified in the directory parameter

Reply:
Got it and agree it is a problem of reading on my side.

The point is that the signature of the method is wrong.
replace:
```
 def get_default_log_path(directory=None, filename=None, root_path=None):
     if directory is None:
-        directory = 'pg_log'
+        directory = '/var/log/pg_log'
```
by
```
 def get_default_log_path(directory='/var/log/pg_log', filename=None, root_path=None):
```

That the Python ways to set a default parameter....

Regards.

Reply:
And you will not need the If directory is None I think.

Reply:
@nhomar Thanks for your comment, I made the changes proposed by you